docs/user/intro/vitepress.rst

```diff
@@ -0,0 +1,60 @@
+VitePress
+=========
+
+.. meta::
+   :description lang=en: Learn how to host VitePress documentation on Read the Docs.
+
+`VitePress`_ is a static site generator with a focus on performance and simplicity.
+
+Minimal configuration is required to build an existing VitePress project on Read the Docs.
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+    version: 2
+
+    build:
+    os: ubuntu-lts-latest
+    tools:
+        nodejs: "latest"
+    jobs:
+        install:
+        - npm install vitepress
+        build:
+        # The site was created by running `vitepress init`
+        # and following the official guide
+        # https://vitepress.dev/guide/getting-started
+        - vitepress build docs
+        - mkdir -p $READTHEDOCS_OUTPUT/
+        - mv docs/.vitepress/dist $READTHEDOCS_OUTPUT/html
+
+.. _VitePress: https://vitepress.dev/
+
+.. button-link:: https://dbtoolsbundle.readthedocs.io/en/stable/
+
+    DbToolsBundle uses VitePress and Read the Docs
+
+Getting started
+---------------
+
+- If you have an existing VitePress project you want to host on Read the Docs, check out our :doc:`/intro/add-project` guide.
+- If you're new to VitePress, check out the official `Getting started with VitePress`_ guide.
+
+.. _Getting started with VitePress: https://vitepress.vuejs.org/guide/getting-started.html
+
+
+Example repository and demo
+---------------------------
+
+Example repository
+    https://github.com/readthedocs/test-builds/tree/vitepress
```

Comment:
I think we should have a separate repo for these types of examples we are linking from our docs. We use test-builds for testing, and has lots of branches that are not relevant/outdated, it's also easy for us to see which branches we need to update if we have a separate repo.

Reply:
Yea, probably -- I'm mostly just trying to get these initial pages setup, but would like to have full repos for these examples so they're easier to find on disk -- perhaps at https://github.com/readthedocs-examples

Reply:
@stsewd if you're doing reviews, it would be useful to :+1: them if you think they're ready :)

Reply:
I'm fine using `test-builds` for now as quick examples. The examples we have in @readthedocs-examples are _very hard_ to create and impossible to maintain with such a small team.

I'd use @readthedocs-examples if we agree on creating very small examples as we are doing with `test-builds` 😄

Reply:
I think a production site using us is the best option, which we have here. I think test-builds is a kind of "minimal example" that works for folks that want to see it. I do worry that example projects are a lot more effort, and probably not a ton of benefit? I think pointing at real users is gonna be way more impactful if people want to see something real.